nuspec/Frends.Csv.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
```

Comment:
This file can actually be removed as it is not used when using the new Project file format